openapi/openapi.json

```diff
@@ -1991,202 +2337,459 @@
+          "schema" : {
+            "type" : "string"
+          }
+        }, {
+          "name" : "after",
+          "in" : "query",
+          "description" : "List of values of fields present in sort param that are used to search after. \n \n> **What it does**: Retrieve the data placed after the pointed element, given the provided order (sort).\n \n> __**Restriction 1**__: **after** param works only combined with **sort** param.\n \n> __**Syntax**__: `after={value1},{value2},...,{valueN} & sort={field1},{field2},...,{fieldN}`.\n \n> **Note 1**: *{value1}` and `{value2}` are the values of `{field1}` and `{field2}` in the last hit returned in the previous search\n \n> **Note 2**: The order of fields and values matters. *{value1},{value2}* must be in the same order of *{field1},{field2}* in **sort** param\n \n> **Note 3**:  The last field `{fieldN}` must be the id field specified in the collection **collection.params.idPath** (returned as **md.id**) and `{valueN}` its corresponding value.\n \n> __**Example**__: *sort=`-date,id` & **after**=`01/02/2019,abcd1234`*. Gets the following hits of the previous search that stopped at date *01/02/2019* and id *abcd1234*.\n \n> __**Restriction 2**__: **from** param must be set to 0 or kept unset\n \n",
+          "schema" : {
+            "type" : "string"
+          }
+        }, {
+          "name" : "before",
+          "in" : "query",
+          "description" : "Same idea that after param, but to retrieve the data placed before the pointed element, given the provided order (sort).",
+          "schema" : {
+            "type" : "string"
+          }
         }, {
           "name" : "max-age-cache",
           "in" : "query",
           "description" : "max-age-cache",
-          "required" : false,
-          "type" : "integer",
-          "format" : "int32"
+          "schema" : {
+            "type" : "integer",
+            "format" : "int32"
+          }
         } ],
         "responses" : {
           "200" : {
             "description" : "Successful operation",
-            "schema" : {
-              "$ref" : "#/definitions/Hits"
+            "content" : {
+              "application/json;charset=utf-8" : {
+                "schema" : {
+                  "type" : "array",
```

Comment:
will be fixed by removing the @array annotation

Reply:
ok